gcloud/bigtable/client.py

```diff
@@ -86,12 +92,22 @@ class Client(_ClientFactoryMixin, _ClientProjectMixin):
                   interact with the Cluster Admin or Table Admin APIs. This
                   requires the :const:`ADMIN_SCOPE`. Defaults to :data:`False`.
 
+    :type user_agent: str
+    :param user_agent: (Optional) The user agent to be used with API request.
+                       Defaults to :const:`DEFAULT_USER_AGENT`.
+
+    :type timeout_seconds: int
+    :param timeout_seconds: Number of seconds for request time-out. If not
+                            passed, defaults to
+                            :const:`DEFAULT_TIMEOUT_SECONDS`.
+
```

Comment:
Yes it is yet-to-be-added.

- `user_agent` gets sent in request metadata
- `timeout_seconds` is a rule to give up on a request if it doesn't succeed (like a socket timeout)